### PR TITLE
make the layout of the web page

### PR DESCRIPTION
**Pull request by kovpaka**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,204 @@
+*,
+::after,
+::before {
+    box-sizing: border-box;
+}
+
+.container-section {
+    max-width: 1170px;
+    margin: auto;
+    text-align: center;
```
Comment (siarheikarabko) [inline style.css:10]: I don't think that this container should control child element positions

Comment (siarheikarabko) [reply style.css:10]: Consider adding left/right padding to the container.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,204 @@
+*,
+::after,
+::before {
+    box-sizing: border-box;
+}
+
+.container-section {
```
Comment (siarheikarabko) [inline style.css:7]: I'd rather name it as `content-container` .

File: style.css [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    max-width: 1170px;
+    margin: auto;
+    text-align: center;
+}
+
+.navbar {
+    display: flex;
+    overflow: hidden;
```
Comment (siarheikarabko) [inline style.css:15]: Why do you need it?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+    min-height: 90px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navbar--color {
```
Comment (siarheikarabko) [inline style.css:21]: Let's discuss modificator using.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,204 @@
+*,
+::after,
+::before {
+    box-sizing: border-box;
+}
+
```
Comment (siarheikarabko) [inline style.css:6]: Also, it's a good idea to set up default typography styles (font styles).

File: style.css [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+    font-size: 32px;
+}
+
+.navbar__links {
+    font-size: 16px;
+}
+
+.banner {
+    min-height: 520px;
+    padding-top: 115px;
```
Comment (siarheikarabko) [inline style.css:35]: Probably padding-bottom is also needed.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+
+.section-text:first-child {
+    padding-top: 40px;
+}
+
+.section-text:last-child {
+    padding-bottom: 50px;
+}
+
+.flex-wrapper {
```
Comment (siarheikarabko) [inline style.css:70]: Lets discuss this class.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+    font-size: 40px;
+}
+
+.banner__additional-text {
+    font-size: 16px;
+    text-transform: uppercase;
+}
+
+.section-text {
+    overflow-wrap: break-word;
```
Comment (siarheikarabko) [inline style.css:51]: Let's discuss this property.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+            <div class="container-section">
+                <h1 class="banner__main-text">Lorem ipsum laro <br> dolore sit amet vivamus</h1>
+                <p class="banner__additional-text">Lorem ipsum laro sit amet dolore sit ame</p>
+            </div>
+        </div>
+    </header>
+
+    <div class="container-section">
+        <div class="section-text">
+            <h1 class="section-text__header">Lorem ipsum dolor sit amet</h1>
```
Comment (siarheikarabko) [inline index.html:33]: The page should contain only single `<h1>`

File: index.html [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+            <h2 class="section-text__description">
+                Donec odio uisque volutpat mattis eros ullam malesuada erat ut turpis.
+            </h2>
+        </div>
+    </div>
+
+    <div class="container-full container--background-gray">
+        <div class="container-section">
+            <div class="cards flex-wrapper">
```
Comment (siarheikarabko) [inline index.html:42]: I don't see these cards.
Moreover, consider using `<ul>` and `<li>`  instead of divs.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+                <h1 class="banner__main-text">Lorem ipsum laro <br> dolore sit amet vivamus</h1>
+                <p class="banner__additional-text">Lorem ipsum laro sit amet dolore sit ame</p>
+            </div>
+        </div>
+    </header>
+
+    <div class="container-section">
+        <div class="section-text">
+            <h1 class="section-text__header">Lorem ipsum dolor sit amet</h1>
+            <h2 class="section-text__description">
```
Comment (siarheikarabko) [inline index.html:34]: Probably it also should be a paragraph or span.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+                Donec odio uisque volutpat mattis eros ullam malesuada erat ut turpis.
+            </h2>
+        </div>
+    </div>
+
+    <div class="container-full container--background-gray">
```
Comment (siarheikarabko) [inline index.html:40]: I haven't found `container-full` styles.

File: index.html (hunk 10 above)
Comment (siarheikarabko) [reply index.html:42]: Also, let's think about a better name than just `cards`/`card`

File: style.css [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+}
+
+.flex-wrapper {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.cards__card {
+    width: 292px;
```
Comment (siarheikarabko) [inline style.css:77]: Fixed width isn't a responsive solution here. Let's discuss. The same for other 'cards'

File: style.css (hunk 5 above)
Comment (kovpaka) [reply style.css:10]: Done.

File: style.css (hunk 3 above)
Comment (kovpaka) [reply style.css]: Deleted.

File: style.css (hunk 1 above)
Comment (kovpaka) [reply style.css]: Fixed.

File: style.css (hunk 2 above)
Comment (kovpaka) [reply style.css]: Ok, fixed.

File: style.css (hunk 4 above)
Comment (kovpaka) [reply style.css]: I deleted almost all modifiers because they're useless.

File: style.css (hunk 7 above)
Comment (kovpaka) [reply style.css]: I deleted this class. All styles of this selector moved to other classes.

File: index.html (hunk 9 above)
Comment (kovpaka) [reply index.html]: Got it. Fixed.

File: index.html (hunk 11 above)
Comment (kovpaka) [reply index.html]: I'm not sure, but I chose h2 to "section-title" class and paragraph to "section-description" class.

File: index.html (hunk 12 above)
Comment (kovpaka) [reply index.html]: There are no "container-full" class anymore.

File: style.css (hunk 13 above)
Comment (kovpaka) [reply style.css]: Fixed.

File: style.css (hunk 6 above)
Comment (kovpaka) [reply style.css]: You're right.

File: index.html (hunk 10 above)
Comment (kovpaka) [reply index.html]: Changed.

**Pull request merged**